docs/pages/application-access/cloud-apis/aws-console.mdx

```diff
@@ -190,39 +399,88 @@ Application Service:
 
 ### Configure the Teleport Application Service
 
-On the host where you will run the Teleport Application Service, create a file
-at `/etc/teleport.yaml` with the following content:
-
-```yaml
-version: v3
-teleport:
-  join_params:
-    token_name: "/tmp/token"
-    method: token
-  proxy_server: "teleport.example.com:443"
-auth_service:
-  enabled: off
-proxy_service:
-  enabled: off
-ssh_service:
-  enabled: off
-app_service:
-  enabled: "yes"
-  apps:
-  - name: "awsconsole"
-  # The public AWS Console is used after authenticating the user from Teleport
-    uri: "https://console.aws.amazon.com/ec2/v2/home"
-```
-
-Edit `/etc/teleport.yaml` to replace `teleport.example.com:443` with the host
-and port of your Teleport Proxy Service or Teleport Cloud tenant, e.g.,
-`mytenant.teleport.sh:443`.
-
-The `app_service` field configures the Teleport Application Service. Each item
-within `app_service.apps` is an application configuration. 
-
-Note that the URI must start with one of the following values in order to be
-recognized as an AWS console:
+1. On the host where you will run the Teleport Application Service, create a
+   file at `/etc/teleport.yaml` with the following content:
+
+   ```yaml
+   version: v3
+   teleport:
+     join_params:
+       token_name: "/tmp/token"
+       method: token
+     proxy_server: "teleport.example.com:443"
+   auth_service:
+     enabled: off
+   proxy_service:
+     enabled: off
+   ssh_service:
+     enabled: off
+   app_service:
+     enabled: "yes"
+     apps:
+     - name: "awsconsole"
+     # The public AWS Console is used after authenticating the user from Teleport
+       uri: "https://console.aws.amazon.com/ec2/v2/home"
+   ```
+
+1. Edit `/etc/teleport.yaml` to replace `teleport.example.com:443` with the host
```

Comment:
can we use the same `in place editor` thingy we do elsewhere in this guide for writing over `teleport.example.com:443`?